Packs/Workday/Integrations/Workday_IAM/Workday_IAM.yml

```diff
@@ -42,18 +42,25 @@ configuration:
   name: first_run
   required: false
   type: 8
-- name: fetch_samples
+- additionalinfo: If checked, the fetch incidents command will sample (at most) five
+    incidents. Use only to sample incidents for classification & mapping.
+  defaultvalue: 'false'
   display: Fetch Samples
+  name: fetch_samples
   required: false
-  defaultvalue: 'false'
-  additionalinfo: If checked, the fetch incidents command will sample (at most) five incidents.
-    Use only to sample incidents for classification & mapping.
   type: 8
+- defaultvalue: '%m/%d/%Y'
+  display: Date Format in Workday Report
+  hidden: false
+  name: workday_date_format
```

Comment:
Do we support all date format? does workday? 
if not we will need to update the mapper to change it for the right format

Reply:
Also let's update the readme file with the new param